### ISSUE TITLE
Several vertical small kana are not 1000-unit “wide”

### ISSUE THREAD

**Issue by RuixiZhang42** [adobe-fonts/source-han-serif#199]
### Prerequisites

* [x] If you are reporting an issue that affects glyphs for characters for a particular region or regions, did you verify that the characters are within the supported scope of the region or regions? This generally means GB 18030 or Tōngyòng Guīfàn Hànzìbiǎo (通用规范汉字表) for China, Big Five or CNS 11643 Planes 1 & 2 for Taiwan, HKSCS-2016 for Hong Kong, the JIS standards for Japan, and KS X 1001 and KS X 1002 for Korea.
* [x] Did you thoroughly search the [open](https://github.com/adobe-fonts/source-han-serif/issues) and [closed](https://github.com/adobe-fonts/source-han-serif/issues?q=is%3Aissue+is%3Aclosed) issues to avoid reporting a duplicate issue?
* [x] Did you go through the [official font readme file](https://github.com/adobe-fonts/source-han-serif/raw/release/SourceHanSerifReadMe.pdf) to better understand the scope of the project, to include the *Known Issues* section at the very end?

### Description

Several vertical small kana have advance that differ from 1000 units. For example, `uni3043.vert` (or CID 65064 in v2.002) in the Medium weight has an advance of 1013 units, among several others. I didn’t check other weights, nor did I know how many glyphs were affected.

**Comment by punchcutter** [comment]
Something got messed up in the ExtraLight source for all small kana except for `asmall-hira.vert`. They are all set to 1020.

**Comment by punchcutter** [comment]
Fixed by 7889f11bf31170b5d092a083b357c8c8130f89e0
```
uni3043-V 1020 -> 1000
uni3045-V 1020 -> 1000
uni3047-V 1020 -> 1000
uni3049-V 1020 -> 1000
uni3063-V 1020 -> 1000
uni3083-V 1020 -> 1000
uni3085-V 1020 -> 1000
uni3087-V 1020 -> 1000
uni308E-V 1020 -> 1000
uni3095-V 1020 -> 1000
uni3096-V 1020 -> 1000
uni30A1-V 1020 -> 1000
uni30A3-V 1020 -> 1000
uni30A5-V 1020 -> 1000
uni30A7-V 1020 -> 1000
uni30A9-V 1020 -> 1000
uni30C3-V 1020 -> 1000
uni30E3-V 1020 -> 1000
uni30E5-V 1020 -> 1000
uni30E7-V 1020 -> 1000
uni30EE-V 1020 -> 1000
uni30F5-V 1020 -> 1000
uni30F6-V 1020 -> 1000
uni31F0-V 1020 -> 1000
uni31F1-V 1020 -> 1000
uni31F2-V 1020 -> 1000
uni31F3-V 1020 -> 1000
uni31F4-V 1020 -> 1000
uni31F5-V 1020 -> 1000
uni31F6-V 1020 -> 1000
uni31F7-V 1020 -> 1000
uni31F8-V 1020 -> 1000
uni31F9-V 1020 -> 1000
uni31FA-V 1020 -> 1000
uni31FB-V 1020 -> 1000
uni31FC-V 1020 -> 1000
uni31FD-V 1020 -> 1000
uni31FE-V 1020 -> 1000
uni31FF-V 1020 -> 1000
uni31F7uni309A-V 1020 -> 1000

```
